lib/jellyfish/notification.ex

```diff
@@ -96,6 +96,28 @@ defmodule Jellyfish.Notification do
           }
   end
 
+  defmodule HlsUploaded do
```

Comment:
In most places we write HLS 

```suggestion
  defmodule HLSUploaded do
```